# 2017-02-06: harry.lin <harry.lin@deltaww.com>
# Support pcie spread spectrum.
--- linux-4.1.8/arch/arm/mach-comcerto/include/mach/serdes-c2000.h.org	2017-01-18 10:54:44.145254586 +0800
+++ linux-4.1.8/arch/arm/mach-comcerto/include/mach/serdes-c2000.h	2017-01-18 10:54:56.508378083 +0800
@@ -1021,15 +1021,15 @@
 	  /* Common CMU block */
   	  { 0x000 << 2, 0x06},
   	  { 0x001 << 2, 0x00},
-  	  { 0x002 << 2, 0x09}, //
+  	  { 0x002 << 2, 0x0d}, //
  	  { 0x003 << 2, 0x00},
  	  { 0x004 << 2, 0x60}, //
  	  { 0x005 << 2, 0x09}, //
- 	  { 0x006 << 2, 0x0e}, //
- 	  { 0x007 << 2, 0x00},
- 	  { 0x008 << 2, 0x00},
+ 	  { 0x006 << 2, 0x7e}, //
+ 	  { 0x007 << 2, 0xff},
+ 	  { 0x008 << 2, 0x03},
  	  { 0x009 << 2, 0x00},
- 	  { 0x00A << 2, 0x00},
+ 	  { 0x00A << 2, 0x0a},
  	  { 0x00B << 2, 0x00},
  	  { 0x00C << 2, 0x00},
  	  { 0x00D << 2, 0x00},
